### PR TITLE
Adds new pegout_confirmed definition

### PR DESCRIPTION
**Pull request by jeremy-then**
Adds new pegout_confirmed definition

### CODE REVIEW

File: rskj-core/src/main/java/co/rsk/peg/utils/BrigeEventLoggerLegacyImpl.java [hunk 1]
```diff
@@ -150,6 +147,11 @@ public void logBatchPegoutCreated(BtcTransaction btcTx, List<Keccak256> rskTxHas
         throw new DeprecatedMethodCallException("Calling BrigeEventLoggerLegacyImpl.logBatchPegoutCreated method after RSKIP146 activation");
     }
 
+    @Override
+    public void logPegoutConfirmed(Sha256Hash btcTxHash, long pegoutCreationRskBlockNumber) {
```
Comment (marcos-iov) [inline rskj-core/src/main/java/co/rsk/peg/utils/BrigeEventLoggerLegacyImpl.java:151]: We should add a test for a call to this method pre RSKIP 146

**Review by marcos-iov — COMMENTED** [review]
Left a minor comment. Solid work, well done!

File: rskj-core/src/main/java/co/rsk/peg/utils/BrigeEventLoggerLegacyImpl.java (hunk 1 above)
Comment (jeremy-then) [reply rskj-core/src/main/java/co/rsk/peg/utils/BrigeEventLoggerLegacyImpl.java:151]: This pr is only about declaring the new event and adding this simple test. More tests for this event will be added in a different story which is about actually emitting this event.

**Comment by kelvinator07** [comment]
Hi @jeremy-then You Should Also Add a Test To BridgeEventLoggerLegacyImplTest

File: rskj-core/src/main/java/co/rsk/peg/utils/BrigeEventLoggerLegacyImpl.java (hunk 1 above)
Comment (marcos-iov) [reply rskj-core/src/main/java/co/rsk/peg/utils/BrigeEventLoggerLegacyImpl.java:151]: That's fine by me as long as you add the details to the corresponding story or create a new one, to make sure we don't forget about this.

It's a simple test anyway, just checking that you get an exception when calling this method pre RSKIP146

**Review by marcos-iov — APPROVED** [review]

**Review by kelvinator07 — APPROVED** [review]

**Pull request merged**